builds/any/rootfs/jessie/common/overlay/etc/mtab.yml

```diff
@@ -17,3 +17,11 @@ mounts:
     mount: ro
     dir: /mnt/onl/boot
     fsck: false
+
+  # ESP (EFI system partition)
+  EFI-BOOT:
+    mount: ro
+    dir: /mnt/onl/efi-boot
```

Comment:
This seems to be mounted by convention at /boot/efi. 
Since this isn't an ONL partition I suggest we move it to /boot/efi for consistency.